.github/workflows/python.yml

```diff
@@ -88,6 +88,16 @@ jobs:
           CIBW_MANYLINUX_X86_64_IMAGE: manylinux2014
           CIBW_BEFORE_BUILD: "yum install -y eigen3-devel assimp-devel libxml2-devel coin-or-Ipopt-devel swig3"
           CIBW_TEST_COMMAND: "python -c 'import idyntree.bindings'"
+          # Note: here we repair the wheel maintaining the linux_x86_64 platform
+          CIBW_REPAIR_WHEEL_COMMAND_LINUX: "auditwheel repair --plat linux_x86_64 -w {dest_dir} {wheel}"
+
+      # Rename the wheel to manylinux2014 (even if it is not) so that it can be uploaded to PyPI
```

Comment:
I hope the PyPI police will not come to look for us.

Reply:
I was not aware of this workaround, I think I read it the first time [here](https://thomastrapp.com/blog/building-a-pypi-package-for-a-modern-cpp-project/). We do the same also in other projects we maintain.

Reply:
It is somehow documented in https://www.python.org/dev/peps/pep-0513/#compilation-of-compliant-wheels.